src/routes/seeds/seeds.route.ts

```diff
@@ -0,0 +1,8 @@
+import express from 'express'
+import { seedDbController } from '../../controllers/seed.controller'
+
+const seedRouter = express.Router()
+
+seedRouter.get('/seed', seedDbController)
```

Comment:
We don't need an endpoint here. Just make this  a script where we can execute using something like `npm run seed`

Reply:
resolved